app/views/alchemy/admin/clipboard/index.html.erb

```diff
@@ -1,18 +1,18 @@
-<%= turbo_frame_tag :clipboard_items do %>
```

Comment:
You removed the turbo frame, which we introduced in #3255

Do the clipboard updates still work?